syntax = "proto2";

message ForceRotateEggsMessage {
    required uint32 rotateCount = 2;
}
